eta/core/data.py

```diff
@@ -531,6 +532,56 @@ def remove_schema(self):
         '''Removes the enforced schema from the container.'''
         self.schema = None
 
+    def get_attrs_with_name(self, name):
+        '''Get a list of all attributes with a given name
+
+        Args:
+            name: the Attribute name
+        Returns:
+            a list of attributes with the given name
+        '''
+        return [attr for attr in self if attr.name == name]
+
+    def get_attr_with_name(self, name):
+        '''Get the single attribute with a given name
+
+        Args:
+            name: the Attribute name
+        Returns:
+            the Attribute
+        Raises:
+            ValueError if there is not exactly one Attribute with the name
+            `name`
+        '''
+        attrs = self.get_attrs_with_name(name)
+        if len(attrs) != 1:
```

Comment:
What do you think about adding a statement
```python
if len(attrs) == 0:
    return None
```
before this line.

Reply:
Well then there's not distinguishing the attr value if the value is `None`, right?

Reply:
Made a PR with a change that should make us both happy: #401